saleor/payment/gateways/dummy/__init__.py

```diff
@@ -19,78 +16,81 @@ def get_client_token(**connection_params):
     return str(uuid.uuid4())
 
 
+def get_template():
+    return 'order/payment/dummy.html'
+
+
 def get_form_class():
     return DummyPaymentForm
 
 
-def authorize(payment: Payment, payment_token: str, **connection_params):
+def process_payment(payment_information: Dict, **connection_params) -> Dict:
+    """Process the whole payment, by calling authorize and capture."""
+    auth_resp = authorize(payment_information, **connection_params)
+    if auth_resp['error']:
```

Comment:
We should use ```is_success``` instead of ```error``` since the gateway may fail actually but unable to provide the errors.